addons/website_sale_loyalty/data/loyalty_demo.xml

```diff
@@ -0,0 +1,87 @@
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_2')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_3')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_4')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_5')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_6')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_7')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_8')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_9')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_10')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_11')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_13')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_14')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_16')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
+        <value eval="ref('website_sale.website_sale_order_17')" />
+        <value eval="ref('loyalty.program')" />
+    </function>
+
+    <function model="sale.order" name="recompute_loyalty_points">
```

Comment:
By multi he meant to do all that in a single call to recompute_Loyalty_points, but you removed everything now, no?
Was that part of the demo in charge of displaying something in the /my/loyalty? Because now it is empty

Reply:
It is intended to be empty. After discussing we actually reached the conclusion that it made sense to consider that the demo loyalty program was added after the existing sale orders were done, and therefore they should have no impact on loyalty points.